src/main/scala-sbt-0.13/compat.scala

```diff
@@ -89,6 +89,14 @@ package internal {
     object `package`
   }
 
+  package util {
+    object `package` {
+      object JLine {
+        val usingTerminal = sbt.JLine.usingTerminal _
```

Comment:
Also I'm not sure if we should do this or not.. it's cool, but maybe we should just stick to declaring it as:

```scala
def usingTerminal[T](f: jline.Terminal => T): T
```